packages/libs/sdk/.env.test

```diff
@@ -0,0 +1 @@
+QUICKNODE_GRAPHQL_CLIENT_SUPPRESS_WARNINGS=true
```

Comment:
Same thing

Reply:
I removed it, if we need this environment variable set we can hardcode it in the test command